src/content/datachannel/datatransfer/index.html

```diff
@@ -46,7 +46,7 @@
 
     <section>
       <div>
-        <input type="number" id="megsToSend" min="1" name="megs" value="1048"/>
+        <input type="number" id="megsToSend" min="1" name="megs" value="128"/> Megs
```

Comment:
I think this should be MB, and should be in a label element. (Can't remember why the progress labels are in divs – had a good reason which I've now forgotten...)

Also (nitpick) I think the button should probably be left aligned above the input and progress element, to match demos like [this](http://webrtc.github.io/samples/src/content/datachannel/basic/). 

So:

    <div id="button">
        <button>...    
    </div>
    <div class="label">Data size: </div>
    <input ...>
    <label for="megsToSend">megabytes</label>

(The input/label content will be displayed on a single line.)

In css/main.css: 

    div#button {
        margin: 0 0 1em 0;
    }

(You can also get rid of the form selector in that file.)